core/scripts/testing/scripts.js

```diff
@@ -113,12 +113,18 @@ const DEFAULT_THEME = 'md';
       // to set the enabled property to 'always'
       if (paletteName === 'dark' && theme.palette?.dark) {
         theme.palette.dark.enabled = 'always';
+      } else if (paletteName === 'high-contrast' && theme.palette?.highContrast) {
+        theme.palette.highContrast.enabled = 'always';
+      } else if (paletteName === 'high-contrast-dark' && theme.palette?.highContrastDark) {
+        theme.palette.highContrastDark.enabled = 'always';
```

Comment:
I had to enable high contrast to be able to test out the removal of steps since the theme used steps. This implementation is subject to change.